uniffi_bindgen/src/bindings/kotlin/templates/WrappedTemplate.kt

```diff
@@ -0,0 +1,41 @@
+object FfiConverterType{{ name }} {
+    {%- match config %}
+    {%- when None %}
+    {#- No wrapper config, just forward all methods to our inner type #}
```

Comment:
I think this `None` branch should collapse down to:

```kt
typealias {{ name|type_name }} = {{ inner|type_name }}

```

followed by an FfiConverter object that fowards on to the inner with `inline` functions.